jdi-light-angular/src/main/java/elements/common/Icon.java

```diff
@@ -0,0 +1,6 @@
+package elements.common;
+
+import com.epam.jdi.light.ui.html.elements.common.Image;
+
+public class Icon extends Image {
```

Comment:
Remove: we already have same class in jdi-light-html

Reply:
Thanks, fixed